harnessed_jobs/rtm_aliveness_power_on/v0/ccs_rtm_aliveness_power_on.py

```diff
@@ -0,0 +1,209 @@
+
+    logger.info("start tstamp: %f", time.time())
+
+    ccs_sub = CcsSubsystems()
+
+    logger.info("Mapping power supply lines to REBs...")
+    power_lines = map_power_lines_to_rebs(ccs_sub)
+
+    logger.info("will attempt to power on and check currents for")
+    for rebid, power_line in power_lines.items():
+        logger.info("  power line %d for REB ID %d", power_line, rebid)
+
+    # Set publishing interval to 0.1s for trending plots.
+    set_monitoring_interval(ccs_sub.ts8, 100)
+    time.sleep(3)
+
+    # This is the order to power on the various channels.
+    named_lines = ('master', 'digital', 'analog', 'clockhi', 'clocklo',
+                   'heater', 'od')
+
+    power_on_ok = True
+    for rebid, power_line in power_lines.items():
+        rebname = 'REB%d' % rebid
+        logger.info("*****************************************************")
+        logger.info("Starting power-on procedure for %s (power line %s)",
+                    rebname, power_line)
+        logger.info("*****************************************************")
+
+        for name in named_lines:
+            try:
+                logger.info("%s: turning on %s power at %s", rebname,
+                            name, time.ctime().split()[3])
+                ccs_sub.rebps.synchCommand(10, "setNamedPowerOn %d %s True"
+                                           % (power_line, name))
+            except java.lang.Exception as eobj:
+                logger.info("%s: failed to turn on current %s!", rebname, name)
+                raise eobj
+
+            # Allow current to settle after powering on.
+            time.sleep(10)
+
+            # Check the channel values:
+            try:
+                if name in channel:
+                    check_values(ccs_sub, rebid, name, *channel[name])
+            except java.lang.Exception as eobj:
+                logger.info("%s: current check failed for %s",
+                            rebname, name)
+                logger.info(eobj.message)
+                power_on_ok = False
+                break
+
+            time.sleep(2)
+
+        logger.info("Turn on REB clock and rail voltages.")
+
+        # Load sensor-specific configurations.
+        if CCSCCDTYPE == 'ITL':
+            ccs_sub.ts8.synchCommand(10, "loadCategories Rafts:itl")
+            logger.info("loaded configurations: Rafts:itl")
+            ccs_sub.ts8.synchCommand(10, "loadCategories RaftsLimits:itl")
+            logger.info("loaded configurations: RaftsLimits:itl")
+        else:
+            ccs_sub.ts8.synchCommand(10, "loadCategories Rafts:e2v")
+            logger.info("loaded configurations: Rafts:e2v")
+            ccs_sub.ts8.synchCommand(10, "loadCategories RaftsLimits:e2v")
+            logger.info("loaded configurations: RaftsLimits:e2v")
+
+        try:
+            command = "powerOn %d" % rebid
+            logger.info(command)
+            logger.info(ccs_sub.ts8.synchCommand(300, command).getResult())
```

Comment:
The output from the powerOn command should be captured and persisted as a separate data product from the rest of the log file output.